packages/@aws-cdk/cx-api/lib/private/flag-modeling.ts

```diff
@@ -45,4 +45,34 @@ export type FlagInfo = FlagInfoBase & (
     /** Describe restoring old behavior or dealing with the change (Markdown) */
     readonly compatibilityWithOldBehaviorMd?: string }
   | { readonly type: FlagType.VisibleContext }
-);
\ No newline at end of file
+);
+
+/**
+ * The magic value that will be substituted at version bump time with the actual
+ * new V2 version.
+ *
+ * Do not import this constant in the `features.ts` file, or the substitution
```

Comment:
❓
Can we have a team tag or marker for such nuggets of wisdom?
Something mentioned in our docs that we could search on?

Reply:
Ah a special "watch this" marker for comments?

Might be a good idea. Feel free to propose something. This particular one is (somewhat) validated by tests as well.